src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newInverseObj = {};
```

Comment:
Avoid using data structure names in variable names, it's really confusing
```suggestion
  const inversedRobot = {};
```